app/src/main/java/org/fossasia/pslab/experimentsetup/SummingJunctionExperiment.java

```diff
@@ -0,0 +1,321 @@
+package org.fossasia.pslab.experimentsetup;
+
+import android.graphics.Color;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TextInputEditText;
+import android.support.design.widget.TextInputLayout;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.afollestad.materialdialogs.DialogAction;
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.github.mikephil.charting.charts.LineChart;
+import com.github.mikephil.charting.data.Entry;
+import com.github.mikephil.charting.data.LineData;
+import com.github.mikephil.charting.data.LineDataSet;
+import com.github.mikephil.charting.interfaces.datasets.ILineDataSet;
+
+import org.fossasia.pslab.R;
+import org.fossasia.pslab.communication.ScienceLab;
+import org.fossasia.pslab.others.ScienceLabCommon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by asitava on 9/8/17.
+ */
+
+public class SummingJunctionExperiment extends Fragment{
+
+    private static final String ERROR_MESSAGE = "Invalid Value";
+    private static final String MINIMUM_VALUE_FREQUENCY = "Frequency should be more than 10Hz";
```

Comment:
Hi! Yes @jithinbp these validation lines are available almost in every experiment. We can create a java file to include all the error messages and parameters as you suggested! :)